src/test/java/rest/gorestcoIn/utils/helpermethods/ApiCall.java

```diff
@@ -0,0 +1,47 @@
+package rest.gorestcoIn.utils.helpermethods;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import io.restassured.specification.RequestSpecification;
+import okhttp3.Request;
+import rest.gorestcoIn.utils.User;
+
+public class ApiCall {
+
+    public static Response post(User user, String endPoint) {
```

Comment:
this method designed specially to post a user, it's not a good approach, your method should have wider usage, so it shouldn't accept a User object as a parameter.